miden-vm/src/main.rs

```diff
@@ -20,9 +22,50 @@ pub(crate) mod utils;
     version,
     rename_all = "kebab-case"
 )]
+#[command(multicall(true))]
+pub struct MidenVmCli {
+    #[command(subcommand)]
+    behavior: Behavior,
+}
+
+impl From<MidenVmCli> for Cli {
+    fn from(value: MidenVmCli) -> Self {
+        match value.behavior {
+            Behavior::MidenVm { cli } => cli,
+            Behavior::External(args) => Cli::parse_from(args).set_external(),
+        }
+    }
+}
+
+#[derive(Debug, Subcommand)]
+#[command(rename_all = "kebab-case")]
+enum Behavior {
+    /// The Miden VM CLI.
+    MidenVm {
+        #[command(flatten)]
```

Comment:
Should we use `#[command(visible_alias = "miden-vm")]` so we can get routed here when called from the right (sym)link?

Reply:
That makes sense, thanks!